doc/source/whatsnew/v0.19.2.txt

```diff
@@ -71,3 +71,15 @@ Bug Fixes
 
 
 - Explicit check in ``to_stata`` and ``StataWriter`` for out-of-range values when writing doubles (:issue:`14618`)
+
+
+
+
+.. _whatsnew_0192.enhancements.other:
+
+Other enhancements
+^^^^^^^^^^^^^^^^^^
+
+- ``pd.merge_asof()`` can take multiple columns in ``by`` parameter and has specialized types (:issue:`13936`)
```

Comment:
Yes, it's to improve performance rather than cast all integer types to int64. I'll add that description to the line.